samples/server/petstore/go-gin-api-server/main.go

```diff
@@ -17,7 +17,7 @@ import (
 	// once you place this file into your project.
 	// For example,
 	//
-	//    sw "github.com/myname/myrepo/go"
+	//sw "github.com/GIT_USER_ID/GIT_REPO_ID/go"
```

Comment:
It would be nice if the generated value fit the current GitHub repo name and path.
```suggestion
	//sw "github.com/github.com/OpenAPITools/openapi-generator/samples/server/petstore/go-gin-api-server"
```

Reply:
These values can be set using --gitUserId and --gitRepoId. I'll see later how we can do with these to generate better samples suggested above.